Libraries/Storage/AsyncStorage.js

```diff
@@ -21,27 +22,58 @@ var RCTAsyncFileStorage = NativeModules.AsyncLocalStorage;
 var RCTAsyncStorage = RCTAsyncRocksDBStorage || RCTAsyncSQLiteStorage || RCTAsyncFileStorage;
 
 /**
- * AsyncStorage is a simple, asynchronous, persistent, key-value storage
+ * @class
```

Comment:
Just curious, why did you have to put `@class` here, but not in something like `AlertIOS` when you updated that?